e2e/specs/st_disabled.spec.js

```diff
@@ -56,10 +56,6 @@ describe("disable widgets", () => {
       cy.get('.stSlider [role="slider"]').first().parent().click();
 
       cy.get(".stMarkdown").should("have.text", "Value 1: 25");
-
-      cy.get(".element-container").each((el, i) => {
```

Comment:
We don't need to take these snapshots since we already take snapshots in the individual widget e2e tests